wc-admin.php

```diff
@@ -49,6 +49,15 @@ function wc_admin_plugins_notice() {
 	printf( '<div class="error"><p>%s</p></div>', $message ); /* WPCS: xss ok. */
 }
 
+/**
+ * Notify users that the plugin needs to be built
+ */
+function wc_admin_build_notice() {
+	echo '<div class="error"><p>';
+	esc_html_e( 'WooCommerce Admin development mode requires files to be built. From the plugin directory, run <code>npm install</code> to install dependencies, <code>npm run build</code> to build the files or <code>npm start</code> to build the files and watch for changes.', 'wc-admin' );
```

Comment:
You are escaping all HTML here, so `<code>` doesn't end up working:

<img width="915" alt="screen shot 2018-12-12 at 3 16 26 pm" src="https://user-images.githubusercontent.com/689165/49896286-eccbfc00-fe20-11e8-99bc-4c082bbb597b.png">

Reply:
Ha, thanks for pointing that out 🤠